app/views/layouts/_index.html.erb

```diff
@@ -42,7 +42,7 @@
       <%= render 'layouts/menu1' %>
   <% elsif current_user.role == 'Employee' %>
       <%= render 'layouts/menu2' %>
-  <% else %>
+  <% elsif current_user.role == 'Student' || 'Parent' %>
```

Comment:
Please consider using the `case` statement of ruby rather than the `elsif` ladder